CONTRIBUTING.md

```diff
@@ -0,0 +1,43 @@
+Contributing to WALA
+====================
+
+WALA welcomes contributions of all kinds and sizes. This includes everything from from simple bug reports to large features.
+
+Workflow
+--------
+
+We love GitHub issues!
+
+For small feature requests, an issue first proposing it for discussion or demo implementation in a PR suffice.
+
+For big features, please open an issue so that we can agree on the direction, and hopefully avoid investing a lot of time on a feature that might need reworking.
+
+Small pull requests for things like typos, bug fixes, etc are always welcome.
+
+CI checks
+---------
+
+Certain checks run on our CI jobs that do not run as part of a standard run of
+`./gradlew build`.  You can run these checks locally with the following command:
+
+```
+./gradlew -PjavaCompiler=ecj linters compileJava compileTestJava
+```
+
+If this command fails, a CI job will fail as well.
+
+DOs and DON'Ts
+--------------
+
+* DO format your code using [Google Java Format](https://github.com/google/google-java-format).  You can do so by running `./gradlew goJF`.  A CI job will fail if your code is not formatted in this way.
```

Comment:
`goJF` → `googleJavaFormat`. Abbreviating Gradle task names is a handy expert feature, but newcomers will find the full names more understandable.